docs/annexes/spdx-lite.md

```diff
@@ -1,131 +1,137 @@
-
-   | # | Property Name | Cardinality | Comments |  
-   |:-:|:--|:--|:--|  
-   | 1 | /Core/Relationship/spdxId			    | 1..1	| |  
-   | 2 | /Core/Relationship/creationInfo		| 1..1	| |  
-   | 3 | /Core/Relationship/from 				| 1..1	| |  
-   | 4 | /Core/Relationship/to 			    	| 1..*	| |  
-   | 5 | /Core/Relationship/relationshipType	| 1..1	| |  
-
-10. For a /SimpleLicensing/LicenseExpression to be conformant with this profile, the following has to hold:  
-
-    | # | Property Name | Cardinality | Comments |  
-    |:-:|:--|:--|:--|  
-    | 1 | /SimpleLicensing/LicenseExpression/spdxId			| 1..1	| |  
-    | 2 | /SimpleLicensing/LicenseExpression/creationInfo		| 1..1	| |  
-    | 3 | /SimpleLicensing/LicenseExpression/licenseExpression	| 1..1	| |  
-    | 4 | /SimpleLicensing/LicenseExpression/licenseListVersion	| 0..1	| |  
-
-11. For a /SimpleLicensing/SimpleLicensingText to be conformant with this profile, the following has to hold:  
-
-    | # | Property Name | Cardinality | Comments |  
-    |:-:|:--|:--|:--|  
-    | 1 | /SimpleLicensing/SimpleLicensingText/spdxId			| 1..1	| |  
-    | 2 | /SimpleLicensing/SimpleLicensingText/creationInfo		| 1..1	| |  
-    | 3 | /SimpleLicensing/SimpleLicensingText/licenseText		| 1..1	| |  
-    | 4 | /SimpleLicensing/SimpleLicensingText/comment		| 0..1	| |  
+# SPDX Lite
+
+## Explanation of the Lite profile
+
+The Lite profile is designed to make it quick and easy to start a Software Bill of Materials
+in situations where a company may have limited capacity for introducing new items into their processes.
+The Lite profile captures the minimum set of information required
+for license compliance in the software supply chain.
+It contains information about the creation of the SBOM,
+package lists with licensing and other related information,
+and their relationships.
+
+All elements in Lite profile are essential for complying with licenses.
+It is easy to use a SPDX document with the Lite profile
+for anyone who does not have enough knowledge about licensing information
+and easy to import license information from former versions of SPDX Lite format files.
+The Lite profile offers the flexibility to be used either alone
+or in combination with other SPDX profiles
+as a SPDX document in the software supply chain.
+
+## Mandatory and recommended properties
+
+The Lite profile specifies that some properties MUST be present
+and some others SHOULD be present, as much as possible.
+
+The following lists collect and present this information
+for every class present in the SPDX data,
+in a concise and easy-to-follow format.
+The lists of properties are in alphabetical order, for easy reference.
+
+### /Core/SpdxDocument
+   * Mandatory
+     1. creationInfo
+     1. element (may be multiple), MUST have at least one /Core/Sbom object
+     1. rootElement (may be multiple), SHOULD be objects of type /Core/Sbom
+     1. spdxId
```

Comment:
```suggestion
### /Core/SpdxDocument

* Mandatory
    1. creationInfo
    1. element (may be multiple), MUST have at least one /Core/Sbom object
    1. rootElement (may be multiple), SHOULD be objects of type /Core/Sbom
    1. spdxId
```

I can see that 3-space and 2-space indent mixes are consistent across the file, so I guess this is intentional and a preferred format to be consume by the spec-parser?

- If it is intended, we may have to documented this syntax later in the future, like the one we used to have in [grammar.md](https://github.com/spdx/spec-parser/blob/main/docs/md_docs/grammar.md). See https://github.com/spdx/spdx-3-model/issues/354
- If it is not, can we use the standard 4-space indent (which is [required for the nested list](https://python-markdown.github.io/#differences)) instead please? I can make the suggestions in all other instances.

Reply:
Thanks, @bact.
The reason I used the 2-space and 3-space indent is due to GitHub's markdown specification, so that GitHub's markdown parser will format it correctly and not clutter the display.
On the other hands, I don't know the details of how it is formatted in the spec-parser, so I would like to hear from @zvr or @goneall, sorry.

Reply:
No problem at all. Markdown parsers behave slightly differently and it's a headache..

Reply:
@NorioKobota then if it's not something specifically required by the spec-parser and if you have no objection, I can reformat it to be a standard 4-space indent for nested list which will work well on both [GitHub Flavoured Markdown](https://github.github.com/gfm/) and on Python-Markdown (used by MkDocs).

Reply:
Yes, I have no objection.
Thank you for your support.

Reply:
@NorioKobota I have made additional suggestions below, please kindly review. Thank you.

(No change in content, only the indents.)

Reply:
Great thanks, @bact.
LGTM.

Reply:
my pleasure